src/main/java/uk/gov/ons/census/fwmtadapter/model/dto/fwmt/FwmtActionInstruction.java

```diff
@@ -29,6 +29,7 @@
   private Integer ceExpectedCapacity;
   private Integer ceActualResponses;
   private Boolean undeliveredAsAddress;
+  private Boolean blankFormReturned;
```

Comment:
oh joy... consistency is for porridge, eh? Nothing we can do about it, but you'd have thought we'd manage to get our terminology the same across the project